db/migrate/20231010162015_add_counter_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddCounterToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :post_count, :integer
```

Comment:
**OPTIONAL** - Please note that it is important that the tables in your database match the ERD, therefore I suggest you rename this column to `posts_counter` as it is in the ERD. Please fix this 👍 👍